README.md

```diff
@@ -179,7 +179,7 @@ type Binding =
 // imported or exported names.
 // The bindings correspond to the equivalent `import` and `export` declarations
 // of an ECMAScript module.
-type ThirdPartyStaticModuleRecord = {
+type SyntheticStaticModuleRecord = {
   bindings?: Array<Binding>,
```

Comment:
```suggestion
  imports?: Array<ImportBinding>,
  exports?: Array<ExportBinding>,
```

Reply:
This is an orthogonal concern. Moddable has vetted this design with a single bindings array. I believe the relative interleaving of `import from` and `export from` matters to make initialization order deterministic, and `export from` could conceptually fit in either the imports or exports bucket spiritually. My preference is not to court confusion.

Reply:
Oh, I didn't notice the init order. It's ok